chainstate/src/interface/chainstate_interface_impl.rs

```diff
@@ -340,4 +342,43 @@ impl<S: BlockchainStorage> ChainstateInterface for ChainstateInterfaceImpl<S> {
             .get_block_id_tree_as_list()
             .map_err(ChainstateError::FailedToReadProperty)
     }
+
+    fn import_bootstrap_stream<'a>(
+        &mut self,
+        reader: std::io::BufReader<Box<dyn std::io::Read + Send + 'a>>,
+    ) -> Result<(), ChainstateError> {
+        let magic_bytes = self.chainstate.chain_config().magic_bytes().to_vec();
+
+        let mut reader = reader;
+
+        // We clone because borrowing with the closure below borrowing
+        let chainstate_config = self.chainstate.chainstate_config().clone();
```

Comment:
maybe instead of cloning the whole config just pass a single field that is actually used inside import?

also comment is confusing

Reply:
I'll fix the comment.

But I don't think it's worth it. The interface is better when passing the whole config. The reason I chose this is because the import operation is way more expensive than this copy operation, so it doesn't really matter.